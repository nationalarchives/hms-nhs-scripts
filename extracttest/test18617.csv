data.text
""
"London,No 10"
"London,No. 21"
"London,no 32"
"London,no. 43"
"London,o 54"
"London,no, 65"
LondonNo 76
London87
"London,98"
London.101
"London,England"
London;England
",England"
"London, No 10"
"London, No. 21"
"London, no 32"
"London, no. 43"
"London, o 54"
"London, no, 65"
London No 76
London 87
"London, 98"
London. 101
"London, England"
London; England
", England"
"London,England"
London;England
",England"
London
Test one two
test One tWo
test a b
test A B
test A. b
test a. b
test of
test Of
test on
test On
test de
test De
test At
test at
test The
test the
test us
test usa
test ss
test sb
test ns
test nb
test na
test ab
test nj
test Upon
test upon
test ...abc
test ...ABC
...abc
...ABC
test [A
test ]A
test {A
test }A
test 'A
test l'orleans
test L'orleans
test ''orleans
test b'orleans
test c-b
test C-B
test
'test
%test
''test
%%test
hill navy
HILL NAVY
test hill navy
test hill test navy test
test hill navy test
test  hill  navy  test
