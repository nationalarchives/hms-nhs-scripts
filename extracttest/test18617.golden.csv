data.text
""
London
London
London
London
London
London
Londonno
London
London
London.
London
London;England
""
London
London
London
London
London
London
London
London
London
London.
London
London; England
""
London
London;England
""
London
Test One Two
Test One Two
Test a B
Test a B
Test A. B
Test A. B
Test of
Test of
Test on
Test on
Test de
Test de
Test at
Test at
Test the
Test the
Test US
Test USA
Test SS
Test SB
Test NS
Test NB
Test NA
Test AB
Test NJ
Test upon
Test upon
Test ...abc
Test ...abc
...abc
...abc
Test [a
Test ]a
Test {a
Test }a
Test 'a
Test L'Orleans
Test L'Orleans
Test ''orleans
Test B'orleans
Test C-B
Test C-B
Test
'Test
%Test
''test
%%Test
